controllers/likedVideo.controller.js

```diff
@@ -0,0 +1,50 @@
+const { LikedVideo } = require('../models/likedVideo.model ');
+
+const getAllLikedVideos = async (req, res) => {
+  try {
+    const { userID } = req.user;
+    const allLikedVideos = await LikedVideo.findById(userID).populate('videos');
+    res.json({ success: true, allLikedVideos });
+  } catch (error) {
+    res.json({ success: false, message: 'Error retrieving liked videos!', errorMessage: error.message });
+  }
+};
+
+const addToLikedVideos = async (req, res) => {
+  try {
+    const { videoID } = req.params;
+    const { userID } = req.user;
+    const user = await LikedVideo.findById(userID);
+
+    if (!user) {
+      const newLikedVideo = new LikedVideo({
+        _id: userID,
+        videos: [{ _id: videoID }],
+      });
```

Comment:
here also, i think the user already exists because liked video document was created with userId in signup controller.